README.md

```diff
@@ -70,26 +70,48 @@ Send Transaction records:
 ## Sequence Diagram
 
 Track Expense Sequence
-![This is the Architectural Diagram](/TrackExpenseSeq.png)
+![This is the sequence Diagram](/TrackExpenseSeq.png)
 
 Send Records Sequence
-![This is the Architectural Diagram](/SendRecordsSeq.png)
+![This is the sequence Diagram](/SendRecordsSeq.png)
+
+## Running the app
+Run backend:
+cd into the back-end folder first, you will need a .env file with the following content: (for ease of grading/testing purpose, we are stroing this secret in README in plain text)
+
+PORT=4000
+DB_URL=mongodb+srv://BBCNews:QEXVgUC6qDmNwzxG@cluster0.iw74dsb.mongodb.net/Bison-Bank-of-Canada?retryWrites=true&w=majority
+TESTDB_URL=mongodb+srv://BBCNews:QEXVgUC6qDmNwzxG@cluster0.iw74dsb.mongodb.net/Test-DB?retryWrites=true&w=majority
+
+This file needs to be under the back-end folder.
+
+Inside the back-end foler, type npm install then npm run to run it
+
+Run frontend:
+cd into the front-end folder and npm run
```

Comment:
Might need to add more to this but I am not too familiar

Reply:
"npm start", not "npm run", and front-end run the same way with back-end. Running front-end should be the same like back-end. However, since we are doing hardcoded for front-end right now, we don't need .env for front-end, but in the future we will need another .env file for front-end